### PR TITLE
better large screen / tablet UI

### PR DESCRIPTION
**Pull request by toddmedema**
Most notable improvement is that now the touch area is the full size of the screen :)

screen shot 2017-11-08 at 19 53 48
screen shot 2017-11-08 at 19 53 25
screen shot 2017-11-08 at 19 53 14

Closes https://github.com/ExpeditionRPG/expedition-app/issues/503
Closes https://github.com/ExpeditionRPG/expedition-app/issues/513

### CODE REVIEW

File: app/Main.tsx [hunk 1]
```diff
@@ -180,6 +180,9 @@ export function init() {
   const window = getWindow();
   const document = getDocument();
 
+  // Before error handling; being offline / not fetching announcements shouldn't display error message
+  getStore().dispatch(fetchAnnouncements());
```
Comment (smartin015) [inline app/Main.tsx]: I think it'd be safer to add a try/catch within fetchAnnouncements() that ensures all errors are suppressed. That way, we don't have to worry about the ordering of this code.

File: app/Style.scss [hunk 2]
```diff
@@ -1134,29 +1141,11 @@ $fontsizemap: (
 
   .app_container {
     position: relative;
-    border-left: $border_desktop;
-    border-right: $border_desktop;
     display: block;
-    overflow: hidden;
-    width: $abswidthmax;
-    height: $absheightmax;
+    width: 100%;
```
Comment (smartin015) [inline app/Style.scss]: Not that I expect things to explode, but... have you tried this out on regular web and portrait-mode android devices?

File: app/Main.tsx (hunk 1 above)
Comment (toddmedema) [reply app/Main.tsx]: So, retried this - and turns out it's not actually an issue / it's a different part of the code throwing that error. See: https://github.com/ExpeditionRPG/expedition-app/issues/513

File: app/Style.scss (hunk 2 above)
Comment (toddmedema) [reply app/Style.scss]: Yep!

**Pull request merged**